ec2/sc-ec2-linux-jumpcloud-workflows.yaml

```diff
@@ -317,7 +311,10 @@ Resources:
       UserData:
         Fn::Base64: !Sub |
           #!/bin/bash
-          /opt/aws/bin/cfn-init -v --stack ${AWS::StackName} --resource LinuxInstance --configsets SetupCfn,SetEnv,SetTags,SetupDocker --region ${AWS::Region}
+          /usr/sbin/useradd -m ssm-user -G docker
```

Comment:
could we add a comment about why this is setup in UserData instead of `AWS::CloudFormation::Init`?  some yahoo, like me, may try to refactor and move it there in the future.